content/english/communities/slack/join.md

```diff
@@ -40,7 +40,8 @@ If you do not identify as Latinx, you will still receive an invitation as a mult
   {{< fields/member >}}
   {{< fields/linkedin >}}
   {{< fields/location >}}
-  {{< fields/latinx_in_tech help="You will get an invite either way." >}}
+  {{< fields/latinx help="You will get an invite either way." >}}
+   {{< fields/tech help="You will get an invite either way." >}}
```

Comment:
we can remove extra space in beginning and also remove help message for tech question

Reply:
updated